schemas/2019-04-01/deploymentTemplate.json

```diff
@@ -143,8 +143,8 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2014-04-01/Microsoft.Insights.json#/resourceDefinitions/webtests" },
                   { "$ref": "https://schema.management.azure.com/schemas/2015-04-01/Microsoft.NotificationHubs.json#/resourceDefinitions/notificationHubs" },
```

Comment:
There shouldn't be any references to Microsoft.NotificationHubs in this file if they have been onboarded.

Reply:
Sorry, I have fixed that.